### DISCUSSION TITLE
Iceberg extension

### DISCUSSION THREAD

**Discussion by ErikJansenIRefact** [duckdb/duckdb#10613] [Ideas]
Who would be interested in building an extension which allows for reading AND writing to Apache Iceberg table format from DuckDB? Keeping the data in Iceberg (instead of DuckDB file storage) has the benefit of being a (more or less) open format. Other engines are now capable of reading the data as well.

Perhaps a better question. Where can we get support / help in achieving this?

**Comment by Tishj** [comment]
https://github.com/duckdb/duckdb_iceberg

**Comment by ErikJansenIRefact** [comment]
Hi Tishj. 

The duckdb_iceberg extension has only read capability. We are looking into writing to iceberg.
Also not sure if the duckdb_iceberg extension takes into account using all the iceberg metadata when scanning the parquet files.

We are just looking for support. Perhaps we should ask DuckDB Labs?

**Comment by hannes** [comment]
Hi @ErikJansenIRefact can you get in touch at hannes@duckdblabs.com please? Cheers!
